shell/platform/fuchsia/flutter/accessibility_bridge.cc

```diff
@@ -223,20 +223,21 @@ void AccessibilityBridge::AddSemanticsNodeUpdate(
   PruneUnreachableNodes();
 
   tree_ptr_->UpdateSemanticNodes(std::move(nodes));
-  tree_ptr_->Commit();
+  // TODO(35718): Implement the callback here.
```

Comment:
For Flutter style, `TODO`s should be:

`// TODO(username): Implement callback here https://link-to-bug`.

But why do we have a todo here? What needs to be implemented?

Reply:
If Semantics manager sees any error while applying a commit, it will call this callback.
Semantics manager will also close the channel for this view. Flutter should re-register this view.
For more information look at semantics_manager.fidl